### PR TITLE
updated intructions for rick & morty app adding bonus material and deployment instructions

### PR DESCRIPTION
**Pull request by Klausstille**

### CODE REVIEW

**Comment by F-Kirchhoff** [comment]
PR for cohort docs: https://github.com/neuefische/web-protocol-template/pull/15

**Comment by F-Kirchhoff** [comment]
Please don't merge yet, I am working on an example solution for project 2

File: sessions/recap-project-1/quiz-app/README.md [hunk 1]
```diff
@@ -66,98 +67,36 @@ Create the profile page in the same way you created the other two pages.
...
+│   └── navigation.css
+├── global.css
+├── styles.css
+├── profile.html
+├── bookmarks.html
+└── index.html
+```
 
-What is **Vercel**? Briefly put, it is a place to host your web applications.
+❗️ Make sure you apply the BEM method to your Quiz App (Bring more structure to your code by using BEM to name all classes!)
```
Comment (NikRoe) [inline sessions/recap-project-1/quiz-app/README.md:95]:
```suggestion
❗️ Make sure you apply the BEM method to your Quiz App (bring more structure to your code by using BEM to name all classes!).
```

File: sessions/recap-project-2/example-implementation/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+- Add a `maxlength` attribute to the form fields
+- Add a display below the form fields to show the amount of characters
+- Use the `input` event to read the `length` of a field's content and calculate and display the
+  result
+- Think of ways to use the same logic for both form fields and to not repeat your code
+
+
+## 5. Bonus
+
+1. When adding a new question card in your newly created form.html; add as well event listeners to the new button and icon which will have the same functionality as described in Task 1.
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/README.md:115]:
```suggestion
1. When adding a new question card in your newly created `form.html`, also add event listeners to the new button and icon which will have the same functionality as described in Task 1.
```

File: sessions/recap-project-2/example-implementation/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+   > ❗️ You DON'T need any loops or querySelectorAll. Just try to grab the individual button and icon directly after adding them to the DOM by giving them unique attributes.
+
+2. Add an event listener to the dark mode toggle button in your profile page which should toggle on a dark or light version of the profile page
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/README.md:119]:
```suggestion
2. Add an event listener to the dark mode toggle button in your profile page which should toggle on a dark or light version of the profile page.
```

File: sessions/recap-project-2/example-implementation/README.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+   > ❗️ You'll need to set some CSS variables as attributes to the <body> element for example
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/README.md:121]:
```suggestion
   > ❗️ You'll need to set some CSS variables as attributes to the <body> element for example.
```

File: sessions/recap-project-2/quiz-app/README.md [hunk 5]
```diff
@@ -106,10 +108,31 @@ users should be informed about the amount of characters left.
   result
 - Think of ways to use the same logic for both form fields and to not repeat your code
 
+
+## 5. Bonus
+
+1. When adding a new question card in your newly created form.html; add as well event listeners to the new button and icon which will have the same functionality as described in Task 1.
```
Comment (NikRoe) [inline sessions/recap-project-2/quiz-app/README.md:115]:
```suggestion
1. When adding a new question card in your newly created `form.html`, also add event listeners to the new button and icon which will have the same functionality as described in Task 1.
```

File: sessions/recap-project-2/quiz-app/README.md [hunk 6]
```diff
@@ -106,10 +108,31 @@ users should be informed about the amount of characters left.
...
+
+   > ❗️ You DON'T need any loops or querySelectorAll. Just try to grab the individual button and icon directly after adding them to the DOM by giving them unique attributes.
+
+2. Add an event listener to the dark mode toggle button in your profile page which should toggle on a dark or light version of the profile page
```
Comment (NikRoe) [inline sessions/recap-project-2/quiz-app/README.md:119]:
```suggestion
2. Add an event listener to the dark mode toggle button in your profile page which should toggle on a dark or light version of the profile page.
```

File: sessions/recap-project-2/quiz-app/README.md [hunk 7]
```diff
@@ -106,10 +108,31 @@ users should be informed about the amount of characters left.
...
+
+   > ❗️ You'll need to set some CSS variables as attributes to the <body> element for example
```
Comment (NikRoe) [inline sessions/recap-project-2/quiz-app/README.md:121]:
```suggestion
   > ❗️ You'll need to set some CSS variables as attributes to the <body> element for example.
```

File: sessions/recap-project-2/quiz-app/README.md [hunk 8]
```diff
@@ -106,10 +108,31 @@ users should be informed about the amount of characters left.
...
+
+   > ❗️ This functionality should only work for the profile page. We will have a look later how to implement dark/light mode on the whole application later in the course.
```
Comment (NikRoe) [inline sessions/recap-project-2/quiz-app/README.md:123]:
```suggestion
   > ❗️ This functionality should only work for the profile page. Later in the course we will have a look at how to implement dark/light mode on the whole application.
```

File: sessions/recap-project-2/example-implementation/bookmarks.html [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+                d="M17 3H7c-1.1 0-2 .9-2 2v16l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
+              />
+            </svg>
+          </a>
+        </li>
+        <li class="navigation__list-item">
+          <a
+            class="navigation__link"
+            href="form.html"
+            aria-label="go to bookmarks page"
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/bookmarks.html:95]:
```suggestion
            aria-label="go to form page"
```

File: sessions/recap-project-2/example-implementation/form.html [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+    <script src="form.js" defer></script>
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__title">Add Question</h1>
+    </header>
+    <main>
+      <form class="question-form" data-js="form">
+        <label class="question-form__input">
+          <span>Question</span>
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/form.html:18]: Does the `span` have a purpose here? 🤔 

```suggestion
         Question
```

File: sessions/recap-project-2/example-implementation/form.html [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+            data-js="question-input"
+          ></textarea>
+          <span
+            class="question-form__character-count"
+            data-js="question-character-count"
+            >0/150</span
+          >
+        </label>
+        <label class="question-form__input">
+          <span>Answer</span>
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/form.html:31]: Same a above, do we need the `span`? 🤔 

```suggestion
          Answer
```

File: sessions/recap-project-2/example-implementation/form.html [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+            data-js="answer-input"
+          ></textarea>
+          <span
+            class="question-form__character-count"
+            data-js="answer-character-count"
+            >0/150</span
+          >
+        </label>
+        <label class="question-form__input">
+          <span>Tag</span>
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/form.html:44]: Same a above, do we need the `span`? 🤔 

```suggestion
          Tag
```

File: sessions/recap-project-2/example-implementation/form.html [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+                d="M17 3H7c-1.1 0-2 .9-2 2v16l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
+              />
+            </svg>
+          </a>
+        </li>
+        <li class="navigation__list-item navigation__list-item--active">
+          <a
+            class="navigation__link"
+            href="form.html"
+            aria-label="go to bookmarks page"
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/form.html:96]:
```suggestion
            aria-label="go to form page"
```

File: sessions/recap-project-2/example-implementation/form.js [hunk 14]
```diff
@@ -0,0 +1,111 @@
+const form = document.querySelector("[data-js=form]");
+const cardContainter = document.querySelector("[data-js=card-container]");
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/form.js:2]:
```suggestion
const cardContainer = document.querySelector("[data-js=card-container]");
```

File: sessions/recap-project-2/example-implementation/form.js [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+  const formElement = event.target;
+
+  const formData = new FormData(formElement);
+  const data = Object.fromEntries(formData);
+
+  const newCard = Card(data);
+  cardContainter.append(newCard);
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/form.js:20]:
```suggestion
  cardContainer.append(newCard);
```

File: sessions/recap-project-2/example-implementation/index.html [hunk 16]
```diff
@@ -0,0 +1,235 @@
...
+                d="M17 3H7c-1.1 0-2 .9-2 2v16l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
+              />
+            </svg>
+          </a>
+        </li>
+        <li class="navigation__list-item">
+          <a
+            class="navigation__link"
+            href="form.html"
+            aria-label="go to bookmarks page"
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/index.html:195]:
```suggestion
            aria-label="go to form page"
```

File: sessions/recap-project-2/example-implementation/profile.html [hunk 17]
```diff
@@ -0,0 +1,158 @@
...
+            height="24"
+            width="24"
+            viewbox="0 0 48 48"
+            fill="currentColor"
+          >
+            <path
+              d="M21.55 31.5q.05-3.6.825-5.25.775-1.65 2.925-3.6 2.1-1.9 3.225-3.525t1.125-3.475q0-2.25-1.5-3.75t-4.2-1.5q-2.6 0-4 1.475T17.9 14.95l-4.2-1.85q1.1-2.95 3.725-5.025T23.95 6q5 0 7.7 2.775t2.7 6.675q0 2.4-1.025 4.35-1.025 1.95-3.275 4.1-2.45 2.35-2.95 3.6t-.55 4Zm2.4 12.5q-1.45 0-2.475-1.025Q20.45 41.95 20.45 40.5q0-1.45 1.025-2.475Q22.5 37 23.95 37q1.45 0 2.475 1.025Q27.45 39.05 27.45 40.5q0 1.45-1.025 2.475Q25.4 44 23.95 44Z"
+            />
+          </svg>
+          <span>12</span>
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/profile.html:46]: Do we need this `span`? 🤔 

```suggestion
          12
```

File: sessions/recap-project-2/example-implementation/profile.html [hunk 18]
```diff
@@ -0,0 +1,158 @@
...
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewbox="0 0 24 24"
+            width="24px"
+            fill="currentColor"
+          >
+            <path d="M0 0h24v24H0V0z" fill="none" />
+            <path d="M17 3H7c-1.1 0-2 .9-2 2v16l7-3 7 3V5c0-1.1-.9-2-2-2z" />
+          </svg>
+          <span>4</span>
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/profile.html:59]: Do we need this `span`? 🤔 

```suggestion
          4
```

File: sessions/recap-project-2/example-implementation/profile.html [hunk 19]
```diff
@@ -0,0 +1,158 @@
...
+                d="M17 3H7c-1.1 0-2 .9-2 2v16l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
+              />
+            </svg>
+          </a>
+        </li>
+        <li class="navigation__list-item">
+          <a
+            class="navigation__link"
+            href="form.html"
+            aria-label="go to bookmarks page"
```
Comment (NikRoe) [inline sessions/recap-project-2/example-implementation/profile.html:118]:
```suggestion
            aria-label="go to form page"
```

File: sessions/recap-project-3/rick-and-morty-app/components/search-bar/search-bar.css [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+  background-color: transparent;
+}
+
+.search-bar__input:focus {
+  outline: none;
+}
+
+.search-bar__icon {
+  width: 20px;
+  /* filter: invert(); */
```
Comment (NikRoe) [inline sessions/recap-project-3/rick-and-morty-app/components/search-bar/search-bar.css:29]: Do we want to keep this comment?

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi there,

nice work so far! Thanks for the effort you put into this 🙌 

I noticed some minor inconsistencies when it comes to ending phrases with full stops, highlighting file names etc.
In the example implementation there is a typo for the container name.
Some of the aria-labels have a misleading text.

Mostly minor stuff.

File: sessions/recap-project-2/example-implementation/profile.html (hunk 17 above)
Comment (F-Kirchhoff) [reply sessions/recap-project-2/example-implementation/profile.html:46]: yes I think we need it, since it should be placed with flexbox next to the svg

**Pull request merged**